x-pack/plugins/secops/public/apps/start_app.tsx

```diff
@@ -15,17 +16,22 @@ import * as euiVars from '@elastic/eui/dist/eui_theme_k6_light.json';
 
 import { AppFrontendLibs } from '../lib/lib';
 import { PageRouter } from '../routes';
+import { createStore } from '../store';
 
 export const startApp = async (libs: AppFrontendLibs) => {
   const history = createHashHistory();
 
+  const store = createStore();
+
   libs.framework.render(
     <EuiErrorBoundary>
-      <ApolloProvider client={libs.apolloClient}>
-        <ThemeProvider theme={{ eui: euiVars }}>
-          <PageRouter history={history} />
-        </ThemeProvider>
-      </ApolloProvider>
+      <ReduxStoreProvider store={store}>
+        <ApolloProvider client={libs.apolloClient}>
+          <ThemeProvider theme={{ eui: euiVars }}>
+            <PageRouter history={history} />
+          </ThemeProvider>
+        </ApolloProvider>
+      </ReduxStoreProvider>
```

Comment:
I haven't mixed Apollo with Redux Store, so @XavierM or someone with experience in this area will have to comment if this looks correct. The code looks clean though.

Reply:
👍